utils/funcexp/func_cast.cpp

```diff
@@ -143,22 +144,16 @@ int64_t Func_cast_signed::getIntVal(Row& row, FunctionParm& parm, bool& isNull,
     case execplan::CalpontSystemCatalog::CHAR:
     case execplan::CalpontSystemCatalog::TEXT:
     {
-      const auto& value = parm[0]->data()->getStrVal(row, isNull);
-
-      if (isNull)
-      {
-        isNull = true;
-        return 0;
-      }
-
-      return atoll(value.str());
+      const auto& str = parm[0]->data()->getStrVal(row, isNull);
```

Comment:
```suggestion
      return parm[0]->data()->getStrVal(row, isNull).toMCSInt64();
```
IMHO return looks more laconic.

Reply:
thanks for the review - what do you mean by laconic?

Reply:
This means IMHO it is better to use less lines/words in the context.

Reply:
For `NullString` and `Decimal` the change is applicable, but for the other numeric types the return value of e.g. `getDoubleValue()` is a `long double` and not the `datatypes::TDouble` datatype on which the `toMCSSInt64Round()` function is defined. 
Therefore we would need to cast first like so:
`return ((datatypes::TDouble) parm[0]->data()->getDoubleVal(row, isNull)).toMCSSInt64Round();`
I think this is hard to read, but wanted to ask if you would still prefer it this way? I'll adapt `NullString` and `Decimal`.

Reply:
@drrtuy

Reply:
Hm, can't you use datatypes::TDouble ctor like this
`return TDouble(parm[0]->data()->getDoubleVal(row, isNull)).toMCSSInt64Round();` ?

Reply:
still personally think that it's a lot of text in one line, but will adapt :)